public/app/features/templating/adhoc_variable.ts

```diff
@@ -3,6 +3,7 @@ import { Variable, assignModelProperties, variableTypes } from './variable';
 
 export class AdhocVariable implements Variable {
   filters: any[];
+  removeFromUrl: number;
```

Comment:
Think skipUrlSync is better, and make it a boolean?   (removeFromUrl  seems like it's an action that removes it already rendered urls)

Reply:
Done everywhere